### PR TITLE
Export tailwind config

### PR DESCRIPTION
**Pull request by andrewfulton**
This will export a tailwind config js file from the tokens in the UI Toolkit figma file. It's not quite a 1:1 with the chakra version - and not sure it will be, but a good number of the tokens/systems are compatible. 

There may be ways to export some higher-level components that are closer in spirit to what we get with chakra, but this depends much closer on how the frontend is implemented in a TW project - eg is this blade, html or still a js project.

### CODE REVIEW

File: src/export/export-tailwind.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+};
+
+// This takes the responsive textStyles that get applied in a system like Chakra
+// and translates them into a set of responsive styles at the various breakpoints
+// defined in the theme.
+export const generateResponsiveTextStyles = (
+  styles: Dictionary<TextVariant>,
+  breakpoints: Breakpoints
+) => {
+  breakpoints = {
```
Comment (andrewfulton) [inline src/export/export-tailwind.ts:46]: This 0px breakpoint is very silly and would like a sensible way to remove it very soon

File: templates/tailwind.config.js.ejs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// AUTO-GENERATED - File generated by figma2theme <%- version %>, manual edits may be overwritten
```
Comment (andrewfulton) [inline templates/tailwind.config.js.ejs:1]: couldn't figure out the prettier config to make it stop autoformatting these ejs files so wierdly.

Comment (spykr) [reply templates/tailwind.config.js.ejs:1]: You can add a `.prettierignore` file to the root with`*.ejs` as the contents to fix this 👍

**Comment by spykr** [comment]
@andrewfulton there's a step in the "test" workflow called "Generate the theme" which is basically a smoke test which ensures the commands run. Can we add the Tailwind command there too?

File: templates/tailwind.config.js.ejs (hunk 2 above)
Comment (andrewfulton) [reply templates/tailwind.config.js.ejs:1]: There already is one with ejs in there

**Comment by spykr** [comment]
Thanks for getting this started @fuldog, I've pushed up a POC based on this with a few changes: https://github.com/PortableStudios/figma2theme/commit/a2d0e8024962de1e0042f8302ea21ca8e9f23954 🙏

**Pull request closed without merging**